lib/apns/state.ex

```diff
@@ -0,0 +1,16 @@
+defmodule APNS.State do
+  def get(options) do
+    {:ok, queue_pid} = APNS.Queue.start_link
```

Comment:
Moving the start link discussion here because the other one is on an outdated diff. Basicaly, why is the queue a separate process? It lives and dies with the main genserver, and you could just have `queue: []` down on line 12 and do just as well.